include/deal.II/lac/lapack_full_matrix.h

```diff
@@ -205,20 +205,31 @@ class LAPACKFullMatrix : public TransposeTable<number>
    *
    * @note Source and destination must not be the same vector.
    *
-   * @note This template only exists for compile-time compatibility with
-   * FullMatrix. Implementation is only available for
-   * <tt>VECTOR=Vector&lt;number&gt;</tt>
+   * @note The template with @tref number2 only exists for compile-time
+   * compatibility with FullMatrix. Only the case @tref number2 = @tref
+   * number is implemented due to limitations in the underlying LAPACK
+   * interface
```

Comment:
Ah, never mind about the linker error -- I see that you provide an implementation.